src/components/ssi/AuthenticationModal/AuthenticationModal.tsx

```diff
@@ -2,7 +2,9 @@ import { Col, Container, Modal, Row } from 'react-bootstrap'
 import React, { Component } from 'react'
 import AuthenticationQR from './AuthenticationQR'
 import { AuthorizationResponsePayload } from '@sphereon/did-auth-siop'
+import Debug from 'debug'
 
+const debug = Debug('ssi:AuthenticationModal')
```

Comment:
```suggestion
const debug = Debug('sphereon:portal:ssi:AuthenticationModal')
```

Reply:
The above allows you to filter on everything from sphereon (so also our low level libs). Everything related to the portal. Everything related to SSI in this portal, or everything related to the Auth Modal.

Morale of the story. Make sure that you follow a hierarchy

Reply:
Fixed!